topology/University of California San Diego/San Diego Supercomputer Center/SDSC-RDS.yaml

```
@@ -136,3 +136,24 @@ Resources:
   # Other resources if you have any...
   # <RESOURCE NAME>:
   # ...
+
+RDS_AUTH_OSDF_CACHE:
+    Active: false
+    Description: OSDF Cache from RDS backed by RDS's Universial Scale Storage service.
```

```suggestion
    Active: true
    Description: OSDF Cache from RDS backed by RDS's Universal Scale Storage service.
```

@DannySaba we may as well mark this as active (I think that it may actually be required) and fix for a presumed typo. If it's not, we can fix this up later!